draft-ietf-masque-connect-ethernet.md

```diff
@@ -372,6 +372,9 @@ such as a kernel. Those responsibilities are beyond the scope of this document,
 and include, but are not limited to, the handling of broadcast packets and
 multicast groups, or the local termination of PAUSE frames.
 
+If an Ethernet proxying endpoint fails to deliver a frame to an underlying
+Ethernet segment, the endpoint SHOULD drop the frame.
```

Comment:
In general the rule of thumb when adding a SHOULD is to also answer the question "in what circumstances would you not follow the SHOULD?". So I'd say either elaborate on that, or switch to MUST.

Reply:
Done.